messaging-client-databus/src/main/java/com/inmobi/messaging/consumer/databus/DatabusConsumer.java

```diff
@@ -125,6 +126,29 @@ protected void initializeConfig(ClientConfig config) throws IOException {
     } else {
       parseClusterNamesAndMigrateCheckpoint(config, rootDirSplits);
     }
+    /*
+    This timer task starts after an initial delay of 30 minutes
```

Comment:
Why do we want to treat initial delay separately? Isn't it sufficient to just have one frequency for all discovery attempts?